### PR TITLE
Mvp

### PR DESCRIPTION
**Pull request by lenzloesche**
- enter your budget data
- data is shown in graph
- you can switch months and years.
- both IST and SOLL is shown in graph and can be entered.

### CODE REVIEW

File: components/FlexDiv/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+const FlexDiv = styled.div`
```
Comment (Lutz-Dietterich) [inline components/FlexDiv/index.js:3]: I think it always has to be Styled at the beginning.
also `const StyledFlexDiv = styled.div`

File: components/InputForm/index.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import StyledButton from "../button";
+import StyledInput from "./Input";
+import StyledGrid from "./Grid";
+import StyledForm from "./Form";
+import React from "react";
```
Comment (Lutz-Dietterich) [inline components/InputForm/index.js:5]: That can go...

File: components/calendar/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <StyledButton onClick={handlePlusMonth}>+</StyledButton>
+      </Banner>
+    </>
+  );
+}
+
+const Heading2 = styled.h2`
+  margin: 0;
+  padding: 0;
+  font-size: 30px;
```
Comment (Lutz-Dietterich) [inline components/calendar/index.js:31]: Hier fehlt überall das Styled am Anfang.

**Review by Lutz-Dietterich — COMMENTED** [review]
Already looks super good! Found a few small things, but nothing major.

File: components/InputForm/index.js (hunk 2 above)
Comment (lenzloesche) [reply components/InputForm/index.js:5]: I need it because of the `React.Fragment` I use. I need the `React.Fragment` to use the `key`prop

**Review by Lutz-Dietterich — APPROVED** [review]

**Pull request merged**